posydon/grids/post_processing.py

```diff
@@ -25,28 +27,45 @@
 ]
 
 
-MODEL = {
-    "mechanism": None,
-    "engine": None,
-    "PISN": "Marchant+19",
-    "ECSN": "Podsiadlowksi+04",
-    "max_neutrino_mass_loss": 0.5,
-    "kick": True,
-    "sigma_kick_CCSN_NS": 265.0,
-    "sigma_kick_CCSN_BH": 265.0,
-    "sigma_kick_ECSN": 20.0,
-    "max_NS_mass": 2.5,
```

Comment:
A question: I haven't seen the kick values in the new version anymore. Is it correct, that we can leave them out or are they missing now?

Reply:
The post-processing script uses the `collapse_star` method of the `step_SN`. Kicks arguments are used only by the `orbital_kick` method which is not part of the `collapse_star` method. When running the `__call__` method of the StepSN object, e.g. during population synthesis, then they will be called one after the other.